src/modules/workspaces/workspaces.controller.spec.ts

```diff
@@ -133,4 +133,110 @@ describe('Workspace Controller', () => {
       );
     });
   });
+
+  describe('GET /:workspaceId/members', () => {
+    const owner = newUser();
+    const workspace = newWorkspace({
+      owner,
+      attributes: { setupCompleted: true },
+    });
+
+    it('When the workspaceId is missing then it should throw BadRequestException', async () => {
+      const workspaceController = jest.spyOn(
+        workspacesController,
+        'getWorkspaceMembers',
+      );
+
+      const expectgetWorkspaceMembers = expect(async () => {
+        await workspacesController.getWorkspaceMembers(null, owner);
+      });
+      expectgetWorkspaceMembers.rejects.toThrow(BadRequestException);
+      expectgetWorkspaceMembers.rejects.toThrow('Invalid workspace ID');
+
+      expect(workspaceController).toHaveBeenCalled();
+    });
+
+    it('When the user is null or empty then it should throw UnauthorizedException', async () => {
+      const workspaceController = jest
+        .spyOn(workspacesController, 'getWorkspaceMembers')
+        .mockImplementation(async (workspaceId: string, user: any) => {
+          if (!user) throw UnauthorizedException;
+          return workspacesUsecases.getWorkspaceMembers(workspaceId, user);
+        });
+
+      const resFailed = workspacesController.getWorkspaceMembers(
+        workspace.id,
+        null,
+      );
+
+      await expect(resFailed).rejects.toThrow();
+      expect(workspaceController).toHaveBeenCalledTimes(1);
```

Comment:
Same here, do not check that the function you are testing is called, of course is called, you have done it explicitly when running the test.